scripts/docker-hub-publish.sh

```diff
@@ -7,6 +7,6 @@ if [[ -z "$1" ]] ; then
     exit 1
 fi
 
-docker build . -t laminar/laminar-node:$1 -t laminar/laminar-node:latest --no-cache
-docker push laminar/laminar-node:$1
-docker push laminar/laminar-node:latest
+docker build . -t laminardev/laminar-node:$1 -t laminardev/laminar-node:latest --no-cache
```

Comment:
can you revert change on this file and add a new `docker-hub-public-dev` script that is similar to this one but also automatically use `git rev-parse --short HEAD` as the tag

Reply:
Will add `docker-hub-public-dev`. But we still need to keep the changes in this file which rename `laminar/*` to `laminardev/*` no?

Reply:
we have both `laminar` and `lamianrdev` and we will use `lamianr` for public released versions, and `lamianrdev` for internal releases

Reply:
I see...